app/controllers/service_requests_controller.rb

```diff
@@ -571,7 +572,7 @@ def send_individual_service_provider_notification(service_request, sub_service_r
     line_items = []
     @service_request.sub_service_requests.each do |ssr|
       if service_provider.identity.is_service_provider?(ssr)
-        line_items << SubServiceRequest.find(ssr).line_items
+        line_items << SubServiceRequest.find(ssr.id).line_items
```

Comment:
Why are we finding the SubServiceRequest when we already have it from the .each iteration? Can't we do line_items << ssr.line_items?